src/components/utils/src/file_system.cc

```diff
@@ -222,6 +222,10 @@ std::string file_system::GetAbsolutePath(const std::string& path) {
   return std::string(abs_path);
 }
 
+bool file_system::IsFileNameValid(const std::string& file_name) {
+  return file_name.end() == std::find(file_name.begin(), file_name.end(), '/');
```

Comment:
@JackLivio we should check all possible cases with the file name which contains some path in order to block the app access for any folder beyond SDL storage folder. Getting access to external folder could be achieved in few ways:
- By specifying '../' in filename
- Putting some softlink in SDL folder which will link to some external folder and then specifying './someLink/filename'

Also we have requirement to check exactly '/' symbol in filenames, so '../' is just one of cases which should be covered.